src/scene/fancyBackground.ts

```diff
@@ -291,8 +291,31 @@ const addImageBackgroundToSvg = async ({
     fancyBackgroundImage.setAttribute("filter", IMAGE_INVERT_FILTER);
   }
 
-  svgRoot.appendChild(fancyBackgroundImage);
+  const clipRect = svgRoot.ownerDocument!.createElementNS(SVG_NS, "rect");
+  clipRect.setAttribute("x", "0");
+  clipRect.setAttribute("y", "0");
+  clipRect.setAttribute("width", `${dimensions.width}`);
+  clipRect.setAttribute("height", `${dimensions.height}`);
+  clipRect.setAttribute("rx", FANCY_BG_BORDER_RADIUS.toString());
+  clipRect.setAttribute("ry", FANCY_BG_BORDER_RADIUS.toString());
+
+  const clipPath = svgRoot.ownerDocument!.createElementNS(SVG_NS, "clipPath");
+  clipPath.setAttribute("id", "fancyBackgroundImageClipPath");
+  clipPath.appendChild(clipRect);
+  clipRect.setAttribute("preserveAspectRatio", "xMidYMid slice");
```

Comment:
not sure what this is for but I added it here as well

Reply:
According to MDN:
> The preserveAspectRatio attribute indicates how an element with a viewBox providing a given aspect ratio must fit into a viewport with a different aspect ratio.

But the element must have `viewBox` to have an effect, so I'll remove it...